docs/user-guide/src/verify/assert_assume.md

```diff
@@ -7,33 +7,50 @@ function (via an assumption).
 
 ## Assertions
 
-The `prusti_assert!` macro instructs Prusti to verify that a certain property
-holds at a specific point within the body of a function. In contrast to the
-`assert!` macro, which only accepts Rust expressions, `prusti_assert!` accepts
-[specification](../syntax.md) expressions as arguments. Therefore, quantifiers
-and `old` expressions are allowed within a call to `prusti_assert!`, as in
-the following example:
-
-```rust,noplaypen
+The macros `prusti_assert!`, `prusti_assert_eq` and `prusti_assert_ne` instruct Prusti to verify that a certain property holds at a specific point within the body of a function. In contrast to the `assert!`, `assert_eq` and `assert_ne` macros, which only accept Rust expressions, the Prusti variants accept [specification](../syntax.md) expressions as arguments. Therefore, [quantifiers](../syntax.md#quantifiers), [`old`](../syntax.md#old-expressions) expressions and other Prusti specification syntax is allowed within a call to `prusti_assert!`, as in the following example:
```

Comment:
Be consistent with the exclamation mark: `prusti_assert_eq!`, etc. (Also in the rest of the file.)